bindings/cpp/src/lib.rs

```diff
@@ -715,6 +747,50 @@ fn restore_wallet(mnemonic: String, password: String) -> Result<Box<Wallet>> {
     Ok(Box::new(Wallet { wallet }))
 }
 
+fn restore_wallet_save_to_securestorage(
+    mnemonic: String,
+    password: String,
+    servicename: String,
+    username: String,
+) -> Result<Box<Wallet>> {
+    let securestorageinfo = SecureStorageWaleltInfo {
+        mnemonic: mnemonic.clone(),
+        password: password.clone(),
+    };
+    let infobytes = bincode::serialize(&securestorageinfo)?;
+    let infohex = hex::encode(infobytes);
```

Comment:
is there a need for bincode if it needs to be a textual representation in the end? why not just `serde_json::to_string(...) / from_str(...)` ?

Reply:
bincode is not necessary, i'll change to textual json